lib/log.js

```diff
@@ -339,7 +340,9 @@ const util = require('util'),
     function createLogFile(options, next) {
         log.info("log#createLogFile");
         if (options.argv.argv.remain.length === 0) {
-            options.argv.argv.remain[0] = path.resolve(_createDefaultFileName());
+            const fileNameSeperator = "_",
+                fileExt = "log";
+            options.argv.argv.remain[0] = path.resolve(createDateFileName(fileNameSeperator, fileExt));
         } else {
             if (path.extname(options.argv.argv.remain[0]) === "") {
                 options.argv.argv.remain[0] = options.argv.argv.remain[0] + ".log";
```

Comment:
can you use "fileExt" instead  of "log"? (It's optional)

Reply:
I've updated it.